content/en/docs/reference/command-line-tools-reference/feature-gates/windows-graceful-node-shutdown.md

```diff
@@ -0,0 +1,21 @@
+---
+title: WindowsGracefulNodeShutdown
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+    toVersion: "1.32"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.33"
```

Comment:
I don't think we want this in here, until we get to the 1.33 release and make a decision on this in the KEP

Reply:
removed